dashboard/src/components/AppView/ChartInfo/ChartInfo.v2.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import { IRelease } from "shared/types";
+import "./ChartInfo.v2.css";
+import ChartUpdateInfo from "./ChartUpdateInfo";
+
+interface IChartInfoProps {
+  app: IRelease;
+}
+
+function ChartInfo({ app }: IChartInfoProps) {
+  const metadata = app.chart && app.chart.metadata;
+  let versionsInfo;
+  let descriptionInfo;
+  if (metadata) {
+    versionsInfo = (
+      <section className="chartinfo-subsection" aria-labelledby="chartinfo-versions">
+        <h5 className="chartinfo-subsection-title" id="chartinfo-versions">
+          Versions
+        </h5>
+        <div>
+          {metadata.appVersion && <div>App Version: {metadata.appVersion}</div>}
+          <span>Chart Version: {metadata.version}</span>
+        </div>
+      </section>
+    );
+    descriptionInfo = (
+      <section className="chartinfo-subsection" aria-labelledby="chartinfo-description">
+        <h5 className="chartinfo-subsection-title" id="chartinfo-description">
+          Description
+        </h5>
+        <span>{metadata.description}</span>
+      </section>
+    );
+  }
+  return (
+    <>
+      {descriptionInfo}
+      {versionsInfo}
+      <ChartUpdateInfo app={app} />
+    </>
+  );
```

Comment:
I'd keep all the JSX code in the same place. In that way, you only need to check the returned value to know the final code you're returning. Just a suggestion.

```suggestion
  return (
    <>
      {metadata && (
        <section className="chartinfo-subsection" aria-labelledby="chartinfo-description">
          <h5 className="chartinfo-subsection-title" id="chartinfo-description">
            Description
          </h5>
          <span>{metadata.description}</span>
        </section>
      )}
      {metadata && (
        <section className="chartinfo-subsection" aria-labelledby="chartinfo-versions">
          <h5 className="chartinfo-subsection-title" id="chartinfo-versions">
            Versions
          </h5>
          <div>
            {metadata.appVersion && <div>App Version: {metadata.appVersion}</div>}
            <span>Chart Version: {metadata.version}</span>
          </div>
        </section>
      )}
      <ChartUpdateInfo app={app} />
    </>
  );
```